.gitignore

```diff
@@ -5,6 +5,7 @@ build/
 !**/src/main/**/build/
 !**/src/test/**/build/
 *.log
+.editorconfig
```

Comment:
This file is created by the DPS gradle spring boot plugin.  We do not want or need to commit it.